Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -97,14 +97,98 @@ public static Command DeleteActionMap(int actionMapIndex)
             };
         }
 
+        public static Command CopyActionMapSelection()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                CopyPasteHelper.CopyActionMap(state);
+                return state;
+            };
+        }
+
+        public static Command CutActionMapSelection()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                CopyPasteHelper.CopyActionMap(state);
+                return DeleteActionMap(state.selectedActionMapIndex).Invoke(state);
+            };
+        }
+
+        public static Command CopyActionBindingSelection()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                CopyPasteHelper.Copy(state);
+                return state;
+            };
+        }
+
+        public static Command CutActionsOrBindings()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                CopyPasteHelper.Copy(state);
+                return state.selectionType == SelectionType.Action ?
+                    DeleteAction(state.selectedActionMapIndex, Selectors.GetSelectedAction(state)?.wrappedProperty.FindPropertyRelative("m_Name").stringValue).Invoke(state)
```

Comment:
I know there is code in many places doing this, but if there is internal viability of "m_Name" (maybe its not here) I definitely think we should use nameof(Type.m_Name) to avoid code like this breaking during future refactoring.